onko_dicom.py

```diff
@@ -67,7 +68,18 @@ def open_file(self):
         if not full_path[0]:
             return
 
-        self.plot_w.plot_dcm(full_path[0])
+        try:
+            logging.log(logging.INFO, "Attempting to graph/open file")
+            self.plot_w.plot_dcm(full_path[0])
+            logging.log(logging.INFO, "successfully opened graph/file")
+
+        except pydicom.errors.InvalidDicomError as e:
+            logging.log(logging.ERROR, f"InvalidDicomError, Missing Dicom Header most likely. Error:({e})")
+        except AttributeError as e:  # very common error, seems to be everything after it has header but not full file.
+            logging.log(logging.ERROR, f"AttributeError, Missing Attribute most likely. Error:({e})")
```

Comment:
use the lazy format otherwise pylint picks up

Reply:
also use err or somthing rather than e as it shows another pylint err

Reply:
Whats the lazy format? Also, will change it from e